examples/arp_response.rs

```diff
@@ -191,6 +188,5 @@ fn make_arp_response_packet(src_mac_addr: &MacAddr, packet: &mut pv::Packet) ->
     arp_req.set_target_hw_addr(dest_mac_addr);
     arp_req.set_target_proto_addr(dest_ipv4_addr);
 
-    /* replace packet data with ARP packet */
-    packet.replace_data(&buffer)
+    packet.end = packet.start + 42; // minimum ARP packet size
```

Comment:
"minimum ARP packet size" is wrong comment for here.

Reply:
And, In the future, `packet.resize(42)` is more preferable

Reply:
There is `packet_size()` methods for each struct and I changed code using it